roles/container-engine/containerd/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Download and unarchive containerd
+  unarchive:
+    src: "{{ containerd_download_url }}"
+    dest: /usr/local
+    remote_src: yes
+
+- name: Download and unarchive crictl
+  unarchive:
+    src: "{{ crictl_download_url }}"
+    dest: /usr/local/bin
+    remote_src: yes
+
+- name: Install crictl config
+  template:
+    src: crictl.yaml.j2
+    dest: /etc/crictl.yaml
+    owner: bin
+    mode: 0644
+
+- name: Install crictl completion
+  shell: /usr/local/bin/crictl completion >/etc/bash_completion.d/crictl
```

Comment:
does this work on all distro's ?

Reply:
I add distro checker beginning of task  like:

```
---
- name: Fail containerd setup if distribution is not supported
  fail:
    msg: "{{ ansible_distribution }} is not supported by containerd."
  when:
    - not ansible_distribution in ["CentOS","RedHat", "Ubuntu", "Debian"]
```